From b8d44d037c4b4ea5abbc3fea6c7753e8c8606ea6 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 03 Oct 2021 23:38:13 +0300
Subject: [PATCH] [SW-52828] bitops: add bit_ull macro

Add the BIT_ULL() macro definition for bits beyond 32 bit.
This is needed to support the boot_if API.

Change-Id: I607098661968a5ea7252b8f8f7198385c479ce90
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/include/bitops.h b/include/bitops.h
index 10df664..30642b3 100644
--- a/include/bitops.h
+++ b/include/bitops.h
@@ -19,6 +19,7 @@
 #endif
 
 #define BIT(nr)		(1 << (nr))
+#define BIT_ULL(nr)	(1ULL << (nr))
 #define BIT_WORD(nr)	((nr) / BITS_PER_LONG)
 
 /* extracts "n" number of bits from position "p" in the given number "v" */
